File: APP_ENGINE/BlueCompass/src/main/java/com/singhtwenty2/OceanVista/data/model/dto/request/RegisterRequestDTO.java
package com.singhtwenty2.OceanVista.data.model.dto.request;

import com.singhtwenty2.OceanVista.data.model.enums.NotificationPreference;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RegisterRequestDTO {
    private String email;
    private String name;
    private String phoneNumber;
    private String password;
    private NotificationPreference notificationPreference;
}

File: NATIVE_ANDROID/app/src/main/java/com/singhtwenty2/oceanvista/feature_auth/data/remote/model/dto/request/RegisterRequestDTO.kt
package com.singhtwenty2.oceanvista.feature_auth.data.remote.model.dto.request

import com.singhtwenty2.oceanvista.feature_auth.data.remote.model.enums.NotificationPrefrance

data class RegisterRequestDTO(
    val email: String,
    val notificationPreference: NotificationPrefrance,
    val password: String
)
